webpack.config.js

```diff
@@ -28,6 +28,7 @@ module.exports = {
       name: '@lightninglabs/lnc-web',
       type: "umd",  // see https://webpack.js.org/configuration/output/#outputlibrarytype
     },
+    globalObject: 'this',
```

Comment:
This flag is needed to support running this package in NodeJS on the server. 
Ref: https://webpack.js.org/configuration/output/#outputglobalobject